src/QsCompiler/CommandLineTool/CommandLineTool.csproj

```diff
@@ -32,6 +32,14 @@
     <ProjectReference Include="..\Compiler\Compiler.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="!$([MSBuild]::IsOsPlatform('Windows'))">
```

Comment:
Why aren't we doing this on Windows?

Reply:
We have an alternate mechanism that works on Windows using the `LlvmLibs` property of the compiler:
https://github.com/microsoft/qsharp-compiler/blob/738f2e4178dd4e0975d654dd7a386fdd6290cdab/src/QsCompiler/CommandLineTool/Options.cs#L157-L172
However, that mechanism doesn't work on Linux or Mac (see this comment in a related issue: https://github.com/dotnet/runtime/issues/8516#issuecomment-315151041). But importantly, while this mechanism of listing the dlls as dependencies would work across all platforms, it introduces a relationship between the two packages that causes the nuget to try and download the LlvmBindings.Native package when loading the cmdline tool, which breaks on platforms that don't have our qdk-alpha feed set up. I stumbled across this in the build pipeline, and had to rely on the fact that skipping the dependency on Windows matches the fact that we only package the nupkg on Windows.

To put it another way, this is purely a dev/test dependency and only required on Linux and Mac, and should not be part of the package we publish.

Reply:
For reference, this is the error that happens if those references are kept in for Windows where the package is built (from [here](https://dev.azure.com/ms-quantum-public/Microsoft%20Quantum%20(public)/_build/results?buildId=37744&view=logs&j=663ee715-dcc5-5b8a-8432-471131753a80&t=f52eef0a-2744-54fe-275f-cf63671504d5&l=398):
```
EXEC : error :  [D:\a\1\s\submodules\qsharp-runtime\src\Simulation\QSharpFoundation\Microsoft.Quantum.QSharp.Foundation.csproj]
    An assembly specified in the application dependencies manifest (qsc.deps.json) was not found:
      package: 'Microsoft.Quantum.LlvmBindings.Native', version: '13.0.0-CI-20220129-005156'
      path: 'runtimes/win-x64/native/libLLVM.dll'
```
That error shows up even though the .NET binaries never refer to that native dependency, because the dotnet pack logic thinks they are required.